.editorconfig

```diff
@@ -155,3 +155,10 @@ csharp_space_after_dot = false
 # Wrapping preferences
 csharp_preserve_single_line_statements = true
 csharp_preserve_single_line_blocks = true
+
+#####################################
+# Analayers Serverity Configuration #
+#####################################
+
+# CS1591: Missing XML comment for publicly visible type or member
+dotnet_diagnostic.CS1591.severity = warning
```

Comment:
Is the problem that this rule is firing for our samples? If so we should stop that from happening.

`.editorconfig` is hierarchical - ex: you can add another `.editorconfig` in the sample directory and just disable the rule there.

Reply:
I added separate editorconfig file into samples directory